tests/schema.rs

```diff
@@ -759,7 +759,8 @@ fn test_root_error_is_not_swallowed_on_parse_error() -> Result<(), String> {
     let raw_schema = r#"/not/a/real/file"#;
     let error = Schema::parse_str(raw_schema).unwrap_err();
 
-    if let Error::JSON(e) = error {
+    // TODO: (#82) this should be a ParseSchemaError wrapping the JSON error
```

Comment:
I don't understand this TODO. In addition, #82  has been closed already.

Reply:
Might be leftover from a rebase.

Reply:
Removed.